code/builders/builder-manager/package.json

```diff
@@ -52,7 +52,7 @@
     "@yarnpkg/esbuild-plugin-pnp": "^3.0.0-rc.10",
     "browser-assert": "^1.2.1",
     "ejs": "^3.1.8",
-    "esbuild": "^18.0.0 || ^19.0.0 || ^0.20.0",
+    "esbuild": "^0.18.0 || ^0.19.0 || ^0.20.0",
```

Comment:
I guess this is the same as `^0.18.0`, isn't it?

Reply:
Not according to https://jubianchi.github.io/semver-check/#/^0.18.0/0.19.0

![Screenshot 2024-03-11 at 10 27 20](https://github.com/storybookjs/storybook/assets/3070389/f26975c4-d3e2-4cfb-a4b9-3c951515ce99)
![Screenshot 2024-03-11 at 10 27 27](https://github.com/storybookjs/storybook/assets/3070389/7614ee50-05e4-4201-b64c-692da0869d5d)

Reply:
Oh, I forgot that versions start with a 0 as major versions behave differently! Thanks for the clarification.